### PR TITLE
We add a new summary figure 

### PR DESCRIPTION
**Pull request by qiyanjun**
### Description of the proposed additions or changes
I propose to have a summary figure connecting different topics covered by the survey. 
The uploaded PDF provides one possible one. We can revise. 

### Suggested reviewers (optional)
@rando2, @cgreene

### CODE REVIEW

**Comment by rando2** [comment]
@qiyanjun this is extremely cool! @nilswellhausen @yemarshall would you be able to give feedback? Here is the direct link: <https://github.com/qiyanjun/covid19-review/blob/master/content/images/Summary.pdf>

**Comment by agitter** [comment]
I think this concept is great.  I can help you add it into the manuscript once the figure is finalized.  We can also commit any figure source files (e.g. an Adobe Illustrator file) into the repository in case it needs to be edited later.

**Comment by nilswellhausen** [comment]
I think we should switch around the arrows from cDNA to the diagnosis since the cDNA is used to run these tests (conceptually that makes more sense to me). I think we could add to the antibody to SARS-CoV2 arrow the term "Neutralizing Antibodies" in red as a Pharmaceutical agent. I can convert this into a high quality illustator file since I worked for a journal and did that all the time. Once I've done that I will upload it here. This makes it fast an easy to adjust other comments since everything is digital.

**Comment by qiyanjun** [comment]
@nilswellhausen please go ahead! You clearly know better than us in figure editing!

**Comment by qiyanjun** [comment]
more figures are added for parts we feel helpful to have a figure type of visualization. All are in PDF format. @rando2 @agitter @nilswellhausen Please help to comment/accept or reject. BTW: are I and my student who made the figures qualified as a coauthor?

**Comment by rando2** [comment]
@qiyanjun This is excellent! Please just add yourselves to <content/metadata.yaml> and you will be added when this is merged in!

**Comment by qiyanjun** [comment]
we are standardizing these figures to make them share the same type of shape/line style/color patterns as the summary.PDF. Will make the commit by tomorrow.

File: content/metadata.yaml [hunk 1]
```diff
@@ -565,3 +565,34 @@ authors:
...
+    email: ys4aj@virginia.edu
+    contributions:
+      - Writing - Figure Drafting
+    affiliations:
+      - Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
+    code of conduct:
+      confirmed: yes
+    coi:
+      string: "None"
+      lastapproved: !!str 2020-?-?
```
Comment (agitter) [inline content/metadata.yaml:582]:
```suggestion
      lastapproved: !!str 2020-07-09
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -565,3 +565,34 @@ authors:
...
+    contributions:
+      - Writing - Original Draft
+      - Writing - Figure Drafting
+    affiliations:
+      - Department of Computer Science, University of Virginia, Charlottesville, VA, United States of America
+    code of conduct:
+      confirmed: yes
+    coi:
+      string: "None"
+      lastapproved: !!str 2020-?-?
```
Comment (agitter) [inline content/metadata.yaml:598]:
```suggestion
      lastapproved: !!str 2020-07-09
```

File: content/metadata.yaml [hunk 3]
```diff
@@ -565,3 +565,34 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-06-23
+-
+    github: kevinsunofficial
+    name: YuCheng Sun
+    initials: YS
+    orcid: 
+    email: ys4aj@virginia.edu
```
Comment (agitter) [inline content/metadata.yaml:573]:
```suggestion
    email: ys4aj@virginia.edu
```

File: content/metadata.yaml (hunk 1 above)
Comment (kevinsunofficial) [reply content/metadata.yaml:582]: Yes. I made these figures and will keep improving them.

**Comment by qiyanjun** [comment]
We have standardized the figures and also add the proper prefix to show their matching Section-IDs.

**Comment by nilswellhausen** [comment]
Hi! Sorry I am running into a github issue. How can I see all the figures that you have made? It might be easiest to send me these figure pdf's per e-mail and I will make them all into illustrator files with nice visualizations (I usually make summary figures in Nature Reviews style because I think it is the most visually appealing) this weekend!

**Comment by rando2** [comment]
> Hi! Sorry I am running into a github issue. How can I see all the figures that you have made? It might be easiest to send me these figure pdf's per e-mail and I will make them all into illustrator files with nice visualizations (I usually make summary figures in Nature Reviews style because I think it is the most visually appealing) this weekend!

@nilswellhausen it's a little tricky, basically you have to go to the "branch" that this commit is from and then look at the content! Here's a link that will take you straight to where the PDFs are: <https://github.com/qiyanjun/covid19-review/tree/master/content/images>

**Comment by SiminaB** [comment]
Really cool summary fig! This isn't my area, but I have a few small comments:
- Title has a typo ("snd" instead of "and")
- Font size should be bigger and a more readable font should be chosen
- I think you can say just "Pharmaceuticals" instead of "Counter pharmaceuticals."

**Comment by qiyanjun** [comment]
@SiminaB will revise! @nilswellhausen will add your suggestion. Can you find the files now?

**Comment by SiminaB** [comment]
Also, check for typos in the other figures and try to have consistency in terms in legends.

**Comment by nilswellhausen** [comment]
I will take care of that @SiminaB :) I can see them now thanks

**Comment by qiyanjun** [comment]
@SiminaB @nilswellhausen we are standardizing the fonts / fixing typos / and finalizing these figures. 

One question though: we add PAC-MAN (one CRISPR based therapeutic (prophylactic) strategy to combat novel coronavirus and influenza) in the summary.PDF 

However, the current version does not include PAC-MAN in the main draft. 
https://www.biorxiv.org/content/10.1101/2020.03.13.991307v1 

@rando2 Do you recommend we remove PAC-MAN from the summary figure? Or keep it? We will have two versions committed though. One with PAC-MAN in and one without. Two files are with clear file naming to distinguish, one with suffix: wCRISPR and another with suffix: woCRISPR. I leave this to you to decide.

**Comment by nilswellhausen** [comment]
I am working on the figures right now. 
Comment on the small molecule drug figure: The upper part of the figure is outside of the human cell you drew in the bottom of the figure. That makes no sense as all the processes in the top part of the figure are also occuring within the human cell. The way you drew the figure it implies that these drugs act outside of a human cell

**Comment by nilswellhausen** [comment]
Addition to that: Remdesivir does not inhibit translation. It inhibits RNA replication. So the arrow with Remdesivir inhibiting translation is wrong.

**Comment by nilswellhausen** [comment]
Untitled-2

This is a little bit of life cycle and drugs inhibiting parts of the life cycle

**Comment by nilswellhausen** [comment]
COVID vaccines

**Comment by qiyanjun** [comment]
@nilswellhausen figures you generated are absolutely superb!!!! We will stop our updates!

**Comment by qiyanjun** [comment]
@nilswellhausen figures you generated are absolutely superb!!!! We will stop our updates!

**Comment by qiyanjun** [comment]
@nilswellhausen What software do you use to generate these figures? Again, they are superb!

**Comment by qiyanjun** [comment]
@nilswellhausen What software do you use to generate these figures? Again, they are superb!

**Comment by nilswellhausen** [comment]
Hi! Thanks, I appreciate the positive feedback! I usually use a mix of BioRender and Adobe Illustrator. Biorender is nice because they already have a lot of cartoons of receptors and cells etc but the overall structuring is so much better in illustrator. Illustrator also allows you to make all forms and figures from scratch whereas in BioRender you kinda have to work with what they provide you.

**Comment by qiyanjun** [comment]
@nilswellhausen many thanks for the great tips!

**Comment by qiyanjun** [comment]
@nilswellhausen many thanks for the great tips!

**Comment by nilswellhausen** [comment]
Summary

This is the summary figure. It is a little crowded so let me know what all of you suggest @rando2 @SiminaB

**Comment by agitter** [comment]
These look fantastic @nilswellhausen!

In the summary figure, the text in the RT-PCR Testing and Serological Testing regions is too small for me to easily read.  For RT-PCR, perhaps both that and CRISPR-Cas could be enlarged?  Serological Testing could also be enlarged if the B cell slides down and to the right a little.

**Comment by nilswellhausen** [comment]
Summary-2
 Is that better @agitter ?

**Comment by nilswellhausen** [comment]
Summary-3
Is that better @agitter ?

**Comment by agitter** [comment]
Yes, definitely better.  Thanks.

There is still a little more space to enlarge the font size for Positive, Negative, Invalid in Serological Testing.

**Comment by nilswellhausen** [comment]
Alright I enlarged the font size for Positive, Negative, Invalid in the Serological testing as well as positive, negative and threshold for the RT PCR.
Summary-4

**Comment by agitter** [comment]
That's great!  Thanks again.

**Comment by RLordan** [comment]
@nilswellhausen @qiyanjun @rando2 .... all of these figures are awesome! Kudos all of you!

**Comment by rando2** [comment]
WOW, this is such an incredible figure. @qiyanjun @kevinsunofficial @nilswellhausen this is absolutely stunning. I'm happy to merge it in whenever you feel it's ready.

**Comment by qiyanjun** [comment]
@nilswellhausen Mind me add you into my forked COVID19-review branch? Or can you share me a link to your PDF files? 
I will name them using the same naming convention in https://github.com/qiyanjun/covid19-review/tree/master/content/images ?

**Comment by qiyanjun** [comment]
@nilswellhausen in the summary figure, how about we also add text "SARS-CoV-2" right under the virus icon?

**Comment by nilswellhausen** [comment]
Yeah I have to add a couple of things to the figure (receptor names, the Virus etc.). ill upload a final version later today and I can also export them as pdf's and put them in here.

**Comment by nilswellhausen** [comment]
Summary-5
Updated Figure

**Comment by rando2** [comment]
@nilswellhausen just let me know when you are done making updates and I will ping everyone on this PR for final review! These are looking great :)

**Comment by nilswellhausen** [comment]
I think I am done!

**Comment by rando2** [comment]
Thanks @nilswellhausen! @qiyanjun @kevinsunofficial @agitter does this look OK to merge?

**Comment by rando2** [comment]
@nilswellhausen did you drag and drop these images into the comment box?  I'm trying to figure out how to update @kevinsunofficial with these figures without risking losing resolution. Maybe could you email them to me at halie [dot] rando [at] pennmedicine.upenn.edu and then I'll try to update the branch?

**Comment by qiyanjun** [comment]
@nilswellhausen possible to put your raw files In a downloadable link? I can then download, standard the file names, then push ... so that all updates are nicely version controlled.. 

Btw: I added your ID as a collaborator in my forked branch.. therefore you can push there directly if you want to.

**Comment by agitter** [comment]
> does this look OK to merge?

Looks great to me!  Let me know if you need help figuring out where to put these files and how to merge them in.  If the figure files are not too large, we can commit both the version that will go in the manuscript (e.g. .svg) and the source file (e.g. Adobe Illustrator .ai file).

**Comment by nilswellhausen** [comment]
Hi! I sent the files to @rando2 via e-mail. Not sure how to upload them here beside the drag and drop thing I did previously.

**Comment by rando2** [comment]
@nilswellhausen I am also not sure how to update the branch in the browser, I'm going to try to do it from the command line... Thank you so much for sending the PDFs and stay tuned!

**Comment by rando2** [comment]
I think if @qiyanjun accepts my [PR](https://github.com/qiyanjun/covid19-review/pull/2), then Nils' images will get added to this PR? This is a little bit more complex than other PRs I've made, so apologies if it is not quite so seamless! I can also create a PR directly from my fork to the greenelab repo if that would be better, but I was hoping I could link these files in with this PR for keeping everything straight!

**Comment by agitter** [comment]
Do we also have the source files for these `.png` figures?  In #347, we're adding the source files in addition to the figures in case the original figure author or another author needs to edit them later.

**Comment by rando2** [comment]
@agitter that makes sense! @nilswellhausen the other PR has .AFdesign files associated with it, but I'm not sure what the raw format would be for your figures?

**Comment by nilswellhausen** [comment]
its PNG. I have the afdesign files on my computer but they are only good for editing.

**Comment by rando2** [comment]
@nilswellhausen Hmm perhaps can you send those to me as well? I definitely understand @agitter's point about having the afdesign files for backup -- I always end up needing svgs from people in the course of manuscript writing so it's probably best to just keep the raw files on github as well!

**Comment by qiyanjun** [comment]
@rando please email me the figures. Let me update! That may make the workflow easier.

**Comment by bansalvi** [comment]
@nilswellhausen @rando2 Just a small suggestion. The current figure looks great. I would just change the shape of TMPRSS2 as currently it looks like a receptor. We used BioRender for protease (TMPRSS2), figure 1 here https://www.biorxiv.org/content/10.1101/2020.05.08.084806v2.full

**Comment by qiyanjun** [comment]
@nilswellhausen possible to export the three figures you drew to PDF format? I believe vector-formed PDF file formats are much better for future use. my email is yanjun@virginia.edu. Thanks.

**Comment by agitter** [comment]
Have we decided where to insert these figures and what to say about them?  Some initial ideas are:
- **1.1 General Background** for `N000-overview.png`
- **4 Therapeutics and Prophylactics** for `N001-LifeCyclePlusDrugs.png`
- **4.3.4.1 Strategies for and challenges to vaccine development** for `N002-Vaccines.png`

**Pull request merged**